defaults/main.yml

```diff
@@ -138,6 +138,17 @@ grafana:
   grafana_user: 'admin'
   grafana_password: 'admin'
 
+localstack:
+  enabled: false
+  docker:
+    cidr: 
+    repo: https://download.docker.com/linux/centos/docker-ce.repo
+  container: 's3-test'
+  data_dir: /tmp/localstack/data
```

Comment:
Do you definitely want this data dir to be /tmp? I suppose it's fine because this is only used for testing, right?

Reply:
hmmm. now that i re-read the docs:

> Set it to /tmp/localstack/data to enable persistence (/tmp/localstack is mounted into the Docker container), leave blank to disable persistence (default)

so i'll comment above the value but leave it empty.